lib/template-loader.php

```diff
@@ -134,68 +134,67 @@ function gutenberg_override_query_template( $template, $type, array $templates =
  * @return int[] A list of template parts IDs for the given block.
  */
 function create_auto_draft_for_template_part_block( $block ) {
-	if ( 'core/template-part' !== $block['blockName'] ) {
-		return array();
-	}
+	$template_part_ids = array();
```

Comment:
What's the alternative? :thinking: It seemed like the most straight-forward way to allow for the [inner blocks recursion to conditionally extend it](https://github.com/WordPress/gutenberg/pull/23050/files?diff=split&w=1#diff-9d2a925bce94bf2c4f5508ddc24e1037R199).

Reply:
I think I just expected it to be added to the array at the end instead of declaring itself as an array, but I can see how that can be more complicated in PHP.